packages/core/src/shared/settings.ts

```diff
@@ -617,7 +618,7 @@ export function fromExtensionManifest<T extends TypeDescriptor & Partial<Section
  * TODO: Settings should be defined in individual extensions, and passed to the
  * core lib as necessary.
```

Comment:
is the TODO still relevant or feasible, given our new strategy?

Reply:
The work required to refactor for this method far outweighs the benefit we would get from it. I'd say its no longer relevant to anything other than a design decision for a new repo or library.